sqlalchemy_filters/models.py

```diff
@@ -36,7 +28,8 @@ def get_query_models(query):
     :returns:
         A dictionary with all the models included in the query.
     """
+    models = [col_desc['entity'] for col_desc in query.column_descriptions]
+    models.extend(mapper.class_ for mapper in query._join_entities)
```

Comment:
Just as a reference, `sqlalchemy_utils` already has a similar function: `get_query_entities` which may support more options when the query is created.
http://sqlalchemy-utils.readthedocs.io/en/latest/_modules/sqlalchemy_utils/functions/orm.html

I thought it was worth mentioning it (just as a reference) since this would add another dependency to the library and we may not need more than this current implementation provides.